Makefile

```diff
@@ -157,7 +157,6 @@ test/e2e/local: cluster/cleanup cluster/cleanup/crds cluster/prepare cluster/pre
 
 
 .PHONY: test/functional
-test/functional:  export SURF_DEBUG_HEADERS=1
```

Comment:
@ciaranRoche was this change pushed intentionally? I think we could still use this debug information for solving flaky tests.

Reply:
@matskiv I found the dumps to add more noise, if there is a failure with the testing-idp we get an additional dump which provides more information over the surf debug header. Happy to add it back in if you want it.

Reply:
Hmm, I think this change doesn't remove it from PROW runs and Nighly pipeline.
Happy to leave this as is.